iosMath.xcodeproj/xcshareddata/xcschemes/iosMath.xcscheme

```diff
@@ -1,129 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
i add it for that  i have developed one math app ，and i  use it for  convert pic， so i think it a little utils